packages/node_modules/@webex/internal-plugin-ediscovery/src/index.js

```diff
@@ -221,6 +192,8 @@ registerInternalPlugin('ediscovery', EDiscovery, {
                     }
 
                     if (reportRequest.emails) {
+                      // store unencrypted emails for ediscovery service to convert to user ids
+                      reportRequest.unencryptedEmails = reportRequest.emails;
```

Comment:
The service will take these emails, convert to user ids, and remove them, ensuring they are never stored in the db